printer_mods/Baltojikale/LoudOwl aka Stabby/README.md

```diff
@@ -0,0 +1,26 @@
+# LoudOwl aka Stabby
+
+<img src="Visual/LoudOwl_2.jpg" width="400px" />
+<img src="Visual/LoudOWL.jpg" width="400px" />
+
+This is alternative part cooling fan solution which uses dual 5015 fan setup for Afterburner Toolhead. Replaces original parts that houses 4020 fan or any other similar type of mod. This mod is more of situational add on for PLA or similar type filaments which might require a lot of cooling (or printing at super high speeds). It is not constant solution as it increases gantry weight.
+
+## Hardware
+* 2x 5015 24v fans
+* 1x M3x20 SCHS (DIN912)
+
+## Printing
+
+Front piece and Logo piece requires supports!!!!!!!! Used recommended settings for Voron parts: 4 perimeters, 5 tops/bottoms, 40% infill. Though it is possible to use less plastic to reduce weight of plastic parts: 3 perimeters, 3 tops bottoms, 25% infill.
+
+## Assembly
+First print out "Cut tool", check fans compatibility, if fit is acceptable remove fans top body part and cut lower part according to “Cut tool”. Insert fans into “Back piece” (adjust fit if needed by cutting additional plastic from lower fan body). Fans should hold firmly in back part. Close them with “front piece” and secure them using “logo piece” using m3x20 SCHS screw. 
+
+<img src="Visual/Using_cut_tool_2.jpg" width="400px" />
+
+### Optional I: 
+Cut excess wires and solder red to red black to black and additional wires of same colour extend them up to your tool head connectors. 
+
+### Optional II:
+Print “Fan decoration” spirals as thin as possible. Model is 0.4mm height but personally suggest to lover to 0.1 or 0.05mm as weight of these pats is enough to make vibrations on fans (which can be felt and most probably will introduce artefacts on prints. Other suggestion is to use “fan decoration” to make stencil sticker and using white paint make decoration.
+
```

Comment:
You have more images in your `Visual` directory, please include them in this README.

Reply:
All pictures were included.